shell/platform/darwin/ios/framework/Source/FlutterKeyboardManager.mm

```diff
@@ -51,21 +50,8 @@ - (void)addSecondaryResponder:(nonnull id<FlutterKeySecondaryResponder>)responde
 }
 
 - (void)dealloc {
-  // It will be destroyed and invalidate its weak pointers
-  // before any other members are destroyed.
-  _weakFactory.reset();
-
   [_primaryResponders removeAllObjects];
   [_secondaryResponders removeAllObjects];
```

Comment:
Why would we need to remove everything from private properties in dealloc? Is there some way we are vending outside references to these arrays?

Reply:
I don't think it would be necessary, this is the only place `-addPrimaryResponder` and `-addSecondaryResponder` are called:

https://github.com/flutter/engine/blob/76a270f6cc31ba3e83cc602f983eb2302dbbe1e8/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm#L823-L831

I'll remove.